src/vos/vos_dtx.c

```diff
@@ -1568,6 +1568,14 @@ vos_dtx_act_reindex(struct vos_container *cont)
 			int				 count;
 
 			dae_df = &dbd->dbd_active_data[i];
+			if (dae_df->dae_flags & DTX_EF_INVALID)
+				continue;
+
+			if (daos_is_zero_dti(&dae_df->dae_xid)) {
+				D_WARN("Skip invalid active DTX entry\n");
+				continue;
+			}
+
```

Comment:
Do you know where these invalid or aborted entries came from? (anything wrong in DAOS-4239 caused these entries?)

Reply:
I do not know, but I am afraid that we hit zero DTX entries during such test. If it can be reproduced, we can assert() here for verification.